### ISSUE TITLE
Update: screens

### ISSUE THREAD

**Issue by svx** [plone/documentation-roboshots#2]
@polyester how can we get the latest and greatest screenshots from Jenkins into this repo ?

**Comment by polyester** [comment]
first: let's set up the correct (same) branching here as for the docs. Haven't quite read/understood the new scheme yet, but it makes sense if this one follows the same naming scheme for versions.

**Comment by svx** [comment]
OK :)

There is not new scheme yet, the issue is still open :)

The idea is:

- master will be rolling aka unstable/testing
- 5.1 or 5? for all 5 content
- 4.3 for the old 4 content
- ...

I am not sure about 5.1 or 5 IMHO I personally would go with just 5, maybe there is a 5.3.4 at some day and then the whole schema would make no sense.

That is why I ting the URL or branch should be 5, like docs.plone.org/5 and if you go there you can read that you are reading the docs for Plone 5.1.x or so.

Like I said I am not sure, it should be something simple, there are pros and cons for all solutions :)

**Comment by polyester** [comment]
OK, I'll try to see if the new Firefox series works over the weekend; haven't tried the webdriver for that yet.

**Comment by svx** [comment]
Cool ! BTW, we decided already I just found that:
```
========
Versions
========

This documentation has versions, just like Plone itself.

The versions are currently matched to the **major** Plone releases:

- version "3" is valid for Plone 3.0 up to 3.3.x
- version "4" is valid for Plone 4.0 up to 4.3.x
- version "5" is valid for Plone 5.0 up to 5.1

In general, the documentation for a version assumes you are using the latest *minor* version.
If features became available during a minor release cycle, for instance features that are available in Plone 4.3 but were not available in Plone 4.2, that is reflected in the text.

We plan to provide an 'unreleased' version link, to allow people to check, proofread and improve the documentation for the upcoming Plone version(s).
```

**Comment by polyester** [comment]
new shots are in a branch "5", which is actually taken from the 5.1 version of the buildout.coredev branch.

I'll replace the ones in "master" with the ones from the 5.2 branch once the testrun is finished on that.

PS: as for browsers, phantomjs (2.1.3, which is the latest) is still by far the best choice for now, Firefox situation is too much in flux, Chrome is uglier. Phantomjs makes correct ones for all but one (where it's reasonably OK, just not perfect)

So, that is good news for automation. I'll try to see if Gil can teach me how to run these on Jenkins then.

**Comment by gforcada** [comment]
@polyester someone called me? 🙂 is there any buildout/README that I can use as a reference to create the jenkins script/job?

**Comment by polyester** [comment]
@gforcada you are like Batman 🦇 😄 

it's in ProductsCMFPlone at https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/tests/robot/robodoc/README.rst

from a standard buildout.coredev buildout one would do
 bin/pybot -v BROWSER:phantomjs src/Products.CMFPlone/Products/CMFPlone/tests/robot/robodoc
to generate all screenshots. Then the generated screenshots are in a subdir _robots, which then goes into this repo.

**Comment by gforcada** [comment]
There you go: https://jenkins.plone.org/job/robotshots/5/robot/report/

**Comment by gforcada** [comment]
@polyester I made the job to run only once per week, is that enough? 🤔

If you go to the _robot link on the link on my previous comment, you will get all the screenshots. Do you want to automatically push them to this repo as well (every week)?

**Comment by polyester** [comment]
@gforcada once a week should be enough.
Thanks!!

automatically pushing would be wonderful, provided that all the tests run through of course. 
Is there a way to get automatically notified when the tests **_don't_** run through?

**Comment by gforcada** [comment]
@polyester yes of course, should you get an email? I will try to work on that (if my laptop does not die *again* 😨 )

**Comment by polyester** [comment]
@gforcada an email to roboshots@plone.org would be great. I'll make a forward for that.

**Comment by gforcada** [comment]
It is already like that, should we leave the automatic commits for another time?

**Comment by gforcada** [comment]
I fixed phantomjs some days ago and the build has been green for a few builds, should we close this, or leave it open? :thinking:

**Comment by polyester** [comment]
The challenge is still how to commit on a succesful build directly to this repo, is it? I'll leave it to you if it's wiser to close this ticket and open a new one, or leave this one open.

It would have to be from current 'master' branch of CMFPlone to 'master' branch of this repo. The screenshots look fine on visual inspection.

**Comment by gforcada** [comment]
ok, no problem, let's keep this one open then. btw. on the meantime, it is safe to push the latest roboshots to this repository? I had a look at the screenshots generated and at least the tinymce pop ups for links and image are not being shown...

**Comment by gforcada** [comment]
@polyester ping :smile: :arrow_up:

**Comment by polyester** [comment]
fixed the popups for links and image with https://github.com/plone/Products.CMFPlone/commit/43f7bb0b8a1fecc09f2e9053cc0df87c6b11837e

the 'table' one is still not correct, but slowly they are improving.
